layouts/partials/openapi/render-object-table.html

```diff
@@ -67,12 +67,18 @@
 
       TODO: support `patternProperties` here.
     */}}
-    {{ if reflect.IsMap .additionalProperties }}
 
+    {{/* Map `additionalProperties: true` to the equivalent `additionalProperties: {}` */}}
```

Comment:
If this change is accepted, the comments above need to be adjusted.

Reply:
This is now obsolete due to https://github.com/matrix-org/matrix-spec/pull/1909/files#r1687890480.